ibc/types.go

```diff
@@ -42,6 +42,8 @@ type ChainConfig struct {
 	EncodingConfig *testutil.TestEncodingConfig
 	// Required when the chain uses the new sub commands for genesis (https://github.com/cosmos/cosmos-sdk/pull/14149)
 	UsingNewGenesisCommand bool `yaml:"using-new-genesis-command"`
+	// Configuration describing additional sidecar processes.
```

Comment:
This covers chain-level sidecars, but can you include an additional slice for per-node sidecars?

Reply:
I was handling all sidcars from this single slice by using the bool field `ValidatorProcess`. the way I imagined this would work is people would instantiate the sidcars and add them to the slice, for any sidecar that should be scoped at a validator level `ValidatorProcess` will be `true`

Chain level sidecars are handled here https://github.com/strangelove-ventures/interchaintest/blob/7d8dff5bc89351f27ead0b24315fd5ed43572ad4/chain/cosmos/cosmos_chain.go#L697-L725

and then the validator sidecars are handled here 
https://github.com/strangelove-ventures/interchaintest/blob/7d8dff5bc89351f27ead0b24315fd5ed43572ad4/chain/cosmos/cosmos_chain.go#L587-L596

im indifferent to the path forward here and can definitely see the benefits of making this more explicit so if you think thats a better design i can make those changes